modules/test/playwright/tests/object-web/objectDefinitions.spec.ts

```diff
@@ -88,4 +90,97 @@ test.describe('Manage object definitions through Model Builder', () => {
 			objectDefinition.id
 		);
 	});
+
+	test('linked object definitions are created when object definitions are related and put into different folders', async ({
+		apiHelpers,
+		modelBuilderPage,
+		page,
+		viewObjectDefinitionsPage,
+	}) => {
+		await page.goto('/');
+
+		const objectFolder =
+			await apiHelpers.objectAdmin.postRandomObjectFolder();
+
+		const objectDefinition1 =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition(
+				objectFolder.externalReferenceCode
+			);
+
+		const objectDefinition2 =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition('default');
+
+		const objectRelationshipLabel =
+			'objectRelationshipLabel' + getRandomInt();
+		const objectRelationshipName =
+			'objectRelationshipName' + objectRelationshipRandomNumber;
+
+		const objectRelationshipData: Partial<ObjectRelationship> = {
+			label: {
+				en_US: objectRelationshipLabel,
+			},
+			name: objectRelationshipName,
+			objectDefinitionExternalReferenceCode1:
+				objectDefinition1.externalReferenceCode,
+			objectDefinitionExternalReferenceCode2:
+				objectDefinition2.externalReferenceCode,
+			objectDefinitionId1: objectDefinition1.id,
+			objectDefinitionId2: objectDefinition2.id,
+			objectDefinitionName2: objectDefinition2.name,
+			type: 'oneToMany' as ObjectRelationshipType,
+		};
+
+		await apiHelpers.objectAdmin.postObjectRelationship(
+			objectRelationshipData
+		);
+
+		await viewObjectDefinitionsPage.goto();
+
+		await viewObjectDefinitionsPage.openObjectFolder('default');
+
+		await viewObjectDefinitionsPage.viewInModelBuilder();
+
+		await expect(
+			modelBuilderPage.objectDefinitionNodes
+				.filter({
+					hasText: objectDefinition1.name,
+				})
+				.locator('svg.lexicon-icon-link')
+		).toBeVisible();
+
+		await expect(
+			modelBuilderPage.objectDefinitionNodes.filter({
+				hasText: objectDefinition2.name,
+			})
+		).toBeVisible();
+
+		await modelBuilderPage.goto({
+			objectFolderName: objectFolder.label['en_US'],
```

Comment:
Hey @rebsilva, we can use object folder name here

Reply:
Futhermore, I would like to hear your opinion about use "Go to folder" button, instead chenge the URL. Do you think it is a important step in the test flow?

Reply:
i think is important, i will change to use the "go to folder" button instead